src/declarations.d.ts

```diff
@@ -0,0 +1,42 @@
+declare module '@onflow/fcl' {
```

Comment:
out of curiosity, I just checked on the status of TS support for fcl, it's a neverending story :D https://github.com/onflow/developer-grants/issues/18